influxql/INFLUXQL.md

```diff
@@ -44,56 +44,123 @@ decimal_digit       = "0" .. "9" .
 
 Database names are more limited than other identifiers because they appear in URLs.
 
+The rules:
+
+- must start with an upper or lowercase ASCII letter
+- may contain only ASCII leters, decimal digits, "_", and "-"
+
 ```
 db_name             = ascii_letter { ascii_letter | decimal_digit | "_" | "-" } .
 ```
 
+#### Examples:
+
+```
+mydb
+MyDB
+my-db_3
+```
+
 ## Identifiers
 
+Identifiers are things like measurement names, retention policy names, tag keys, etc.
+
+The rules:
+
+- double quoted identifiers can contain any unicode character other than a new line
+- unquoted identifiers must start with an upper or lowercase ASCII character
+- unquoted identifiers may contain only ASCII letters, decimal digits, "_", and "."
```

Comment:
can unquoted identifiers actually contain `.`? I thought those needed to be quoted because of the db, rp, name ident splitting.